react/src/component/AuthView.jsx

```diff
@@ -554,6 +557,18 @@ export default class AuthView extends BaseView {
         }).catch(this.snackBarError)
     }
 
+    downloadAgentConfigFile(agentId) {
+        axios.get(`/api/agent/downloadAgentConfigFile/${agentId}`).then(res => {
+            if (res.data.code === 200) {
+                this.setState({
+                    snackbarIsShown: true,
+                    snackbarSeverity: "success",
+                    snackbarMessage: "Agent config file downloaded"
+                })
+            }
+        }).catch(this.snackBarError)
```

Comment:
Just FYI. If you want to download file by axios, you need to set the responseType as 'blob'. 
https://stackoverflow.com/questions/41938718/how-to-download-files-using-axios